README.md

```diff
@@ -101,37 +127,24 @@ $ cp deployments/cni-conf.json /etc/cni/net.d/
 $ kubectl create -f deployments/crdnetwork.yaml
 ```
 
-### Build SRIOV CNI
+#### Network Object CRDs
 
-1. Compile SRIOV-CNI (dev/k8s-deviceid-model branch):
-```
-$ git clone https://github.com/intel/sriov-cni.git
-$ cd sriov-cni
-$ make
-$ cp build/sriov /opt/cni/bin
-```
-2. Create the SRIOV Network CRD
+Multus uses Custom Resource Definitions(CRDs) for defining additional network attachements. These network attachment CRDs follow the standards defined by K8s Network Plumbing Working Group(NPWG). Please refer to [Multus documentation](https://github.com/intel/multus-cni/blob/master/README.md) for more information.
+1. Create the SRIOV Network CRD
 ```
 $ kubectl create -f deployments/sriov-crd.yaml
 ```
 
-### Build and run SRIOV network device plugin
+#### DANM
+This section explains an example deployment of SRIOV Network device plugin in Kubernetes with DANM.
 
- 1. Clone the sriov-network-device-plugin
- ```
-$ git clone https://github.com/intel/sriov-network-device-plugin.git
-$ cd sriov-network-device-plugin
- ```  
- 2. Build executable binary using `make` 
- ``` 
-$ make
-```      
-> On successful build the `sriovdp` executable can be found in `./build` directory. It is recommended to run the plugin in a container or K8s Pod. The follow on steps cover how to build and run the Docker image of the plugin.
+#### Install DANM 
+Refer to [DANM documentation](https://github.com/nokia/danm#getting-started) for detailed instructions.
 
- 3. Build docker image
- ```
-$ make image
-``` 
+#### Create SR-IOV type networks
```

Comment:
Is this similar to using sriov-cni to create network attachments?

Reply:
it is a CRD based API if that's what you are asking. so in a sense, it is similar, yes.
it is different API though. we don't expose the specifics of each and every CNI backend as Multus does, instead we provide one, generic, abstract API

as above, the necessary manifests showing off how one can create an SR-IOV type network is linked to the README enhancement